frontend/src/i18n/locales/index.ts

```diff
@@ -0,0 +1,9 @@
+import { ResourceLanguage } from 'i18next'
+
+import enSG from './en-sg'
+
+const locales = {
```

Comment:
supernit: prefer  `export const` over `export default`, but am ok! (rationale is in case we rename things, but also cognisant that translation files probably will never be renamed haha)

Reply:
done